boost-maven/boost-boosters/booster-beanValidation20/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>boost.boosters</groupId>
+    <artifactId>beanValidation</artifactId>
```

Comment:
'beanValidation' =>  'bean-validation' or even just 'validation' (as with the API).    Kinda like 'bean-validation'